packages/orchestration/src/utils/registry.js

```diff
@@ -95,6 +96,7 @@ export const convertChainInfo = async registry => {
       stakingTokens: chain.staking?.staking_tokens,
       // UNTIL https://github.com/Agoric/agoric-sdk/issues/9326
       icqEnabled: chain.chain_name === 'osmosis',
+      icaEnabled: icaEnabled(chain.chain_name),
```

Comment:
the same name is being used for a property and a query function. please rename the query function.

Reply:
Renamed